public/components/metrics/sidebar/data_source_picker.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiComboBox, EuiTitle } from '@elastic/eui';
+import React from 'react';
+import { DATASOURCE_OPTIONS } from '../../../../common/constants/metrics';
+import { OptionType } from '../../../../common/types/metrics';
+
+interface DataSourcePickerMenuProps {
+  selectedDataSource: OptionType[];
+  setSelectedDataSource: (sources: OptionType[]) => void;
+}
+
+export const DataSourcePicker = ({
+  selectedDataSource,
+  setSelectedDataSource,
+}: DataSourcePickerMenuProps) => {
+  // const { selectedDataSource, setSelectedDataSource } = DataSourcePickerMenuProps;
+
+  const onChange = (
+    // eslint-disable-next-line no-shadow
+    selectedDataSource
```

Comment:
follow lint suggestions than disabling it?

Reply:
removed this too